cmd/client/main.go

```diff
@@ -119,18 +121,20 @@ func withArgs(args []string, outf outFn, errorf errorFn) {
 	rootCmd.AddCommand(reportCmd(rootArgs, outf, errorf))
 	rootCmd.AddCommand(quotaCmd(rootArgs, outf, errorf))
 
-	if err := rootCmd.Execute(); err != nil {
-		errorf(err.Error())
-	}
+	return rootCmd
 }
 
 func main() {
-	withArgs(os.Args[1:],
+	rootCmd := GetRootCmd(os.Args[1:],
 		func(format string, a ...interface{}) {
 			fmt.Printf(format, a...)
 		},
 		func(format string, a ...interface{}) {
-			fmt.Fprintf(os.Stderr, format, a...)
-			os.Exit(1)
+			fmt.Fprintf(os.Stderr, format+"\n", a...)
+			os.Exit(-1)
```

Comment:
I switched to -1 since I saw this in a few cobra examples I read over the weekend. I just did a quick search to see if there is a general prefered return value to indicate failure and nothing emerged from that. 0 means success and != means failure. So -1 or 1 seem the same in the end.